CHANGELOG.md

```diff
@@ -1,14 +1,27 @@
 # Change Log
 
-## Unreleased Features, Changes & Bugfixes
-**REQUIRED: SF build 2519+ & GP build 4.0.0.332+ OPTIONAL: Nucleus version 1.0.1+**
-- None
+## Upcoming/Unreleased Features, Changes & Bugfixes
+**REQUIRED: SF build 2558+ & GP build 4.0.0.415+ OPTIONAL: Nucleus version 1.1.3+**
+- Added Void World Generation capabilities
+- Added Nether portal fix
```

Comment:
???did you mean the nether portal did not work?

Reply:
Don't worry about the Unreleased/Upcoming changes as none of this is set in stone. But what's happening is people are being sent to nether portals they can't access because it's on an island they don't have access to.

Reply:
Didn't realize you had changed this, will have to check it out soon

Reply:
Those things aren't in this PR, they're stashed on my PC while I address issues with it.

Reply:
Oh,  duh. Let me know if you'd like for me to test changes to nether portals. I had some issues with those in the past that I'd like to get fixed.

Reply:
I could probably sneak in half of the plan ed nether portal changes I want to make available in this build tbh.